.pre-commit-config.yaml

```diff
@@ -1,41 +1,50 @@
+ci:
+  autofix_prs: false
+
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: check-merge-conflict
+      - id: check-case-conflict
+      - id: check-ast
+      - id: debug-statements
+      - id: check-toml
+      - id: check-json
+
+  - repo: https://github.com/pre-commit/pygrep-hooks
+    rev: v1.9.0
+    hooks:
+      # - id: python-check-blanket-noqa # XXX: disallow blanket # noqa
+      - id: python-check-blanket-type-ignore
+      - id: python-check-mock-methods
+      # - id: python-use-type-annotations # XXX: disallow type comments
+
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        types: [python] # FIXME: pyi support that doesn't conflcit with black
```

Comment:
Currently getting an odd bug where `isort` tries to add two blank lines after imports in `.pyi` files, contrary to both typeshed and black conventions. If I can (later) bisect this properly I'll open an issue on isort.

Reply:
I believe this is an isort regression, there is a historic version where this issue was resolved to align with the black profile.

See: https://github.com/PyCQA/isort/issues/1284

Reply:
Yeah, came across that as well. Pinged the maintainer -- for now I'm just going to leave the FIXME.